theano/tensor/nnet/abstract_conv.py

```diff
@@ -1396,7 +1397,7 @@ class BaseAbstractConv(Op):
 
     def __init__(self, convdim,
                  imshp=None, kshp=None, border_mode="valid",
-                 subsample=None, filter_flip=True, filter_dilation=None):
+                 subsample=None, filter_flip=True, filter_dilation=None, num_groups=2):
```

Comment:
Here, the default value is different then at other places. This isn't good. The default should be the same everywhere expect in rare condition. It should also default to the old behavior, like without groups